tests/ut/codegen/test_orchestration_codegen.py

```
@@ -4158,6 +4166,49 @@ def main(self, x: pl.Tensor[[64], pl.FP32]) -> pl.Tensor[[64], pl.FP32]:
+            @pl.function(type=pl.FunctionType.InCore)
+            def consume(self, x: pl.Tensor[[64], pl.FP32]) -> pl.Tensor[[64], pl.FP32]:
+                return x
+
+            @pl.function(type=pl.FunctionType.Orchestration)
+            def main(
+                self,
+                scratch: pl.Tensor[[64], pl.FP32],
+                other: pl.Tensor[[64], pl.FP32],
+            ) -> pl.Tensor[[64], pl.FP32]:
+                with pl.manual_scope():
+                    produced, producer_tid = pl.submit(self.fill, scratch)
```

Ruff (0.15.15)

[warning] Unpacked variable `producer_tid` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)